### ISSUE TITLE
Move List to GameLogic.cs

### ISSUE THREAD

**Issue by mayisa87** [andulica/SlotMachineExercise#43]
https://github.com/andulica/SlotMachineExercise/blob/2878fd9e3ce1924208aa1701d89794279fd45841/Program.cs#L8

As you only use it to pass it to GameLogic anyway, or to UI methods (which won't be necessary if you do further refactoring), move it to GameLogic class.
You only will need a method "public int getListCount()",  for this line of code in Program.cs 

`credits = credits + winnings - linesToPlay.Count;`

Read through other issues too, as they are related to each other

**Comment by andulica** [comment]
https://github.com/andulica/SlotMachineExercise/blob/f807799a9130e2d56d16f377811e6841b83c5f95/Program.cs#L21-L22

An empty list is needed to populate it with user input (lines to play) . I don't see how getListCount method could help me in this particular example

**Comment by mayisa87** [comment]
The idea was, if you refactor your ChooseLines method anyway, to move most part to the GameLogic class, that the list can be in this class too. 

But maybe, first start to refactor the method, by spliting up in separate methods, do it step by step and we/you will see, if it would be easier to keep list in Program.cs or to move it later on to GameLogic.

**Comment by andulica** [comment]
Now I understand. Basically all the params need to be in methods in GameLogic  class and from there to call them in Program.cs. The reason again is to increase the readability and to make it easier to debug code which make sense

**Comment by mayisa87** [comment]
Yes exactly :)

**Comment by andulica** [comment]
I really don't see how I can get rid of all this variables : 
https://github.com/andulica/SlotMachineExercise/blob/a1d1324f5dad1a1aba7faedbb1b69c11b38cb392/Program.cs#L8-L12
Do you have any suggestions ? A tip at least  ,please?

**Comment by andulica** [comment]
One possible solution might be to move these variables to GameLogic and then call them in Program.cs but will it be a valid solution?

**Comment by mayisa87** [comment]
Let me have a look to your current version first. It was only about the list. I'll have a closer look to it and see if it's still best solution as you did a lot of changes so far. Will come back to you on this!

**Comment by mayisa87** [comment]
So I had a look to the code. Just to clarify, it is completely OK to have the linesToPlay in the Program.cs. If you use it there, and maybe also use it more often, that's fine. You have your grid there too.
My suggestion was just because I saw, that you did not use the variable linesToPlay very often. That's why I suggested it.  But as we are still using static calls and no objects, maybe stick to the code like it is. Remove the linesTopLay from GameLogic and let's keep it in Program.cs

Sometimes the view changes after doing some refactoring ;) Hope this is OK to you. 


So remove static list from Game Logic and keep it in Program, that's fine!
